security/try-pod-to-pod-encryption.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Try out the pod-to-pod encryption tech preview
+description: Try out enabling WireGuard for state-of-the-art cryptographic security between pods for Calico clusters.
+---
+
+### Big picture
+
+Enable WireGuard to secure pod-to-pod traffic in a {{site.prodname}} cluster.
+
+> **Warning!** pod-to-pod encryption is a tech preview and should not be used in production clusters. It has had very limited testing and it will contain bugs (please report these on the Calico Users slack or Github).  In addition, it does not support all the features of {{site.prodname}}.
```

Comment:
> **Warning!** Pod-to-pod...

Reply:
We should be upfront about which issues need to be resolved so that a user does not waste time trying out something that we know is not yet supported.  @bmckercher123 - would that be best done here, or in the Before you begin section, or both?

We should let the user know that:
-  Not yet compatible with overlay networks (in particular node ports do not work)
-  Using node ports from within a non-host networked pod will not work. Using the cluster IP does work though.

Reply:
@robbrockbank can you add a list here what you want to be included in not-supported or may-contain-bugs? eBPF mentions a few following `it will contain bugs e.g. `

Reply:
Also - I thought we were going to have some "troubleshooting" section to see if wireguard is actually enabled on a node?  Does that belong in this doc or do we now have separate troubleshooting docs?

Reply:
Not sure. @bmckercher123 can you suggest if a troubleshooting section belongs here or requires a new document?

Reply:
nit addressed. Waiting on guidance on further changes.

Reply:
We should add any troubleshooting right in the doc. Either inline as a step, or add a section at the end. Either works.

Reply:
Updated warning and added troubleshooting section.